### PR TITLE
Update plugins

### PR DESCRIPTION
**Pull request by alxsap**
Trying to fix https://github.com/SAP/stewardci-jenkinsfilerunner-image/issues/89

### CODE REVIEW

**Comment by alxsap** [comment]
Still has Exception https://github.com/SAP/stewardci-jenkinsfilerunner-image/issues/89

**Comment by alxsap** [comment]
ES plugin fix being prepared here https://github.com/SAP/elasticsearch-logs-plugin/pull/43

**Comment by alxsap** [comment]
Basic pipeline works, reference pipeline(s) fail with:

```
java.lang.NoSuchMethodError: No such DSL method 'lock' found among steps [archive, bat, build,...]
```

File: jenkinsfile-runner-base-image/packager-config.yml [hunk 1]
```diff
@@ -55,7 +55,7 @@ plugins:
     artifactId: "pipeline-elasticsearch-logs"
     source:
       git: "https://github.com/SAP/elasticsearch-logs-plugin.git"
-      commit: "0.10.0"
+      commit: "dcbe0ef0780c42aaf078a522c55c74640b4dd8ff" #TODO: released commit - is from https://github.com/SAP/elasticsearch-logs-plugin/pull/43
```
Comment (alxsap) [inline jenkinsfile-runner-base-image/packager-config.yml:58]:
```suggestion
      commit: "77c2c1932156cdcfce7d5dcfaf8f14b9e950ba55" #TODO: released commit - is from https://github.com/SAP/elasticsearch-logs-plugin/pull/43
```

**Comment by alxsap** [comment]
[87f78e7](https://github.com/SAP/stewardci-jenkinsfilerunner-image/pull/90/commits/87f78e7bd5c327f3757f5916f05db6ce1e0e803b) validated successfully

File: jenkinsfile-runner-base-image/packager-config.yml [hunk 2]
```diff
@@ -55,7 +55,7 @@ plugins:
     artifactId: "pipeline-elasticsearch-logs"
     source:
       git: "https://github.com/SAP/elasticsearch-logs-plugin.git"
-      commit: "0.10.0"
+      commit: "77c2c1932156cdcfce7d5dcfaf8f14b9e950ba55" #TODO: released commit - is from https://github.com/SAP/elasticsearch-logs-plugin/pull/43
```
Comment (alxsap) [inline jenkinsfile-runner-base-image/packager-config.yml:58]:
```suggestion
      commit: "0.11.0"
```

**Comment by romanisb** [comment]
Would it make sense to "re-update" the plugin versions again?

**Comment by romanisb** [comment]
Would it make sense to "re-update" the plugin versions again?

**Comment by alxsap** [comment]
> Would it make sense to "re-update" the plugin versions again?

I can do it, I just thought about getting things through the door before breaking anything else again.

**Review by romanisb — APPROVED** [review]

**Comment by alxsap** [comment]
Validation successful

**Pull request merged**